product_analytic/models/account_move.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 Akretion (http://www.akretion.com/) - Alexis de Lattre
+# Copyright 2016 Antiun Ingeniería S.L. - Javier Iniesta
+# Copyright 2017 Tecnativa - Luis Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+INV_TYPE_MAP = {
+    "out_invoice": "income",
+    "out_refund": "income",
+    "in_invoice": "expense",
+    "in_refund": "expense",
+}
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    @api.onchange("product_id")
+    def _onchange_product_id(self):
+        res = super()._onchange_product_id()
+        inv_type = self.move_id.type
+        if self.product_id:
```

Comment:
Hi there,
a few days ago a FIX-PR was made for version 12 in this line, see https://github.com/OCA/account-analytic/pull/283/

Can you add it here too? Thanks a lot! :)

Reply:
@marylla Thank you. I fixed it.